cypress/e2e/webTables.cy.js

```diff
@@ -1,5 +1,171 @@
 describe('Web Tables page', () => {
-  it('', () => {
+  const testData = {
+    firstname: 'Kendrick',
+    lastname: 'Lamar',
+    email: 'lamar.k@qa.team',
+    age: 35,
+    salary: 20000,
+    department: 'QA'
+  };
 
+  beforeEach(() => {
+    cy.visit('/');
+  });
+
+  it('should be a pagination in the table', () => {
+    cy.get('.-pagination')
+      .should('exist');
+
+    cy.get('.-pageInfo')
+      .should('exist');
+
+    cy.contains('.-previous', 'Previous')
+      .should('exist');
+
+    cy.contains('.-next', 'Next')
+      .should('exist');
+  });
+
+  it('should select rows number per page', () => {
+    cy.get('[aria-label="rows per page"]')
+      .select('5 rows');
+
+    cy.get('select')
+      .should('contain', '5 rows');
+  });
+
+  it('should allow to add new employee', () => {
+    cy.get('#addNewRecordButton')
+      .click();
+
+    cy.get('.modal-content')
+      .should('exist');
+
+    cy.findByPlaceholder('First Name')
+      .type(testData.firstname);
+
+    cy.findByPlaceholder('Last Name')
+      .type(testData.lastname);
+
+    cy.get('#userEmail')
+      .type(testData.email);
+
+    cy.findByPlaceholder('Age')
+      .type(testData.age);
+
+    cy.findByPlaceholder('Salary')
+      .type(testData.salary);
+
+    cy.findByPlaceholder('Department')
+      .type(testData.department);
+
+    cy.get('#submit')
+      .click();
+
+    cy.get('[role="rowgroup" ]')
+      .should('contain', testData.firstname)
+      .and('contain', testData.lastname)
+      .and('contain', testData.email)
+      .and('contain', testData.age)
+      .and('contain', testData.salary)
+      .and('contain', testData.department);
+  });
+
+  it('should allow to delete employee', () => {
+    cy.get('#delete-record-3')
+      .should('exist');
+
+    cy.get('#delete-record-3')
```

Comment:
Add an assertion to check if the employee was deleted successfully after clicking on the delete button.